src/commands/attest/create.ts

```diff
@@ -13,6 +13,12 @@ const MAX_SEARCH_RESULTS = 5
 const prompt = promptSync()
 
 export default class CreateAttestations extends Command {
+    static usage = 'attest create -o <attestation-file> [-i <hdf-json> -t <json | xlsx | yml | yaml>]'
+
+    static description = 'Create attestation files for use with `saf attest apply`'
+
+    static examples = ['saf attest create -o attestation.json -i hdf.json']
```

Comment:
Should we add an example without the input file because that is optional?

Reply:
done